hrjobcontract/CRM/Hrjobcontract/DAO/HRJobLeave.php

```diff
@@ -197,6 +211,7 @@ static function &fieldKeys()
                 'id' => 'id',
                 'leave_type' => 'hrjobcontract_leave_leave_type',
                 'leave_amount' => 'hrjobcontract_leave_leave_amount',
+                'add_public_holidays' => 'hrjobcontract_leave_add_public_holidays',
```

Comment:
Seems legit, but fieldKeys() method description says: "Returns an array containing, for each field, the arary key used for that field in self::$_fields" so it would mean that the array values should be equal to the self::$_fields keys.